src/config/config.js

```diff
@@ -8,4 +11,8 @@ export default {
     errorFile: 'logs/error.log',
     logFile: 'logs/combined.log'
   },
+
+  mode: {
+    environment: process.env.NODE_ENVIRONMENT || 'development'
```

Comment:
Looks good here. Are we able to remove that extra space on line 14 and consider using `process.env.NODE_ENV` as it is the standard way to set the node environment.

Reply:
ok will do that

Reply:
Thanks!